### ISSUE TITLE
Turns off once a day around the same time

### ISSUE THREAD

**Issue by ys27** [creativecreatorormaybenot/wakelock#126]
I am using this in my Flutter project and running the app on Android.

I recently updated the versions on Dart (2.8.4) and Flutter (2.2.3) and kept the wakelock version to ^0.1.4+1, which was working perfectly before. After the updates to Dart and Flutter, I noticed that the screen was turning off once a day, between 7-8pm EST. So I updated the wakelock version to ^0.5.2, but it still turns off around the same time. It doesn't matter what time I turn it back on, it always turns off between 7-pm EST.

Any idea why this might be happening? I'm not sure where to even start debugging. Any leads would be great.
If there are any other information that could help, please let me know.

Thank you.

**Comment by creativecreatorormaybenot** [comment]
Hi @ys27, thanks for using the plugin :)

Can you rephrase what your use case is, if you can provide a sample, etc.? It confuses me that you seem to have the app running 24/7.

**Comment by ys27** [comment]
Hello,

I am using it in a clock application, jsut something I have running on my unused tablet. 

Here is a link to the repo: https://github.com/ys27/AlwaysOnClock. It is being used in lib/pages/clockPage.dart

**Comment by creativecreatorormaybenot** [comment]
@ys27 Okay, that is an interesting use case 😅 

I would assume this is some internal system feature that causes this. My suggestion would be for you to try to do the following:

* Call `Wakelock.enable()` every minute / every hour / whatever suits you best
* In your case, this can likely be combined with the clock updating logic 👍 
* Let me know if this solves the problem for you 👌 

The plugin implementation is optimized to deal with repeated calls, so I was actually also fine calling the on every paint call (so 60 times per second) 🙃

**Comment by ys27** [comment]
Thanks for the solid lead, I made the change and I'll report back on what happens tonight!

**Comment by creativecreatorormaybenot** [comment]
@ys27 thanks! 🚀

**Comment by ys27** [comment]
I think that worked! I'll have to keep on eye on it for a few days to really make sure, but that prob was it. Thanks for the quick solution and for the plugin!!
